website/thunderbird/115.0/whatsnew/index.html

```diff
@@ -0,0 +1,289 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% set active_page = "whatsnew" %}
+{% set hide_header = True %}
+{% extends "includes/base-resp.html" %}
+
+{% block page_css %}<link href="/media/css/115release.css" rel="stylesheet" type="text/css" />{% endblock %}
+
+{% block page_title %}{{ _('What’s New in Thunderbird 115') }}{% endblock %}
+
+{% block site_header %}
+  <header>
+    <div class="container">
+      <h1>
+        {{_('What’s New in
+        <br>
+        <span class="tb-title">Thunderbird</span>
+        <span class="version">115</span>')}}
```

Comment:
Let's add another element here with the work "supernova" and make it visually hidden so the whole header can be read by screen readers